modules/core/src/main/java/org/apache/ignite/IgniteSystemProperties.java

```diff
@@ -53,30 +55,43 @@
      * @deprecated Use {@link #IGNITE_LOG_INSTANCE_NAME}.
      */
     @Deprecated
+    @IgniteSystemProperty(description = "If this system property is present the Ignite will include grid name into " +
+        "verbose log.", type = String.class)
     public static final String IGNITE_LOG_GRID_NAME = "IGNITE_LOG_GRID_NAME";
 
     /**
      * If this system property is present the Ignite will include instance name into verbose log.
      */
+    @IgniteSystemProperty(description = "If this system property is present the Ignite will include instance name " +
+        "into verbose log.", type = String.class)
     public static final String IGNITE_LOG_INSTANCE_NAME = "IGNITE_LOG_INSTANCE_NAME";
 
     /**
      * This property is used internally to pass an exit code to loader when
      * Ignite instance is being restarted.
      */
+    @IgniteSystemProperty(description = "This property is used internally to pass an exit code to loader when Ignite " +
```

Comment:
Please, remove all boilerplate text like "This property", "System property".
Also, please, use the spellchecker to ensure correct english in the description.